websockify/websocketproxy.py

```diff
@@ -418,6 +425,12 @@ def websockify_init():
             help="file of concatenated certificates of authorities trusted "
             "for validating clients (only effective with --verify-client). "
             "If omitted, system default list of CAs is used.")
+    parser.add_option("--ssl-version", choices=["tlsv1_1", "tlsv1_2"],
```

Comment:
We should be exhaustive here and allow older stuff in case people need it. And the default should probably be to let the `ssl` module keep its defaults.

(Python also supports TLS 1.3 now, so you might also want to add that)

Reply:
TLS 1.3 :+1: 
default: :+1:

But I'm a little hesitant to explicitly add options for versions that are considered insecure (TLS 1.0 and below). I think providing (potentially insecure) default + three secure options (TLS 1.1, 1.2 and 1.3) would make it both flexible and give users no option to make an unintentional mistake.

BTW, WebSocket has been standardized in 2011, while TLS 1.1 in 2006. I would expect that anyone using WebSockets has a software stack capable of running modern crypto.